packages/app/src/cli/models/extensions/specifications/payments_app_extension.ts

```diff
@@ -0,0 +1,49 @@
+import {createExtensionSpecification} from '../specification.js'
+import {BaseSchema} from '../schemas.js'
+import {zod} from '@shopify/cli-kit/node/schema'
+
+export type PaymentsAppExtensionConfigType = zod.infer<typeof PaymentsAppExtensionSchema>
+const PaymentsAppExtensionSchema = BaseSchema.extend({
+  payment_session_url: zod.string().url(),
+  refund_session_url: zod.string().url(),
+  capture_session_url: zod.string().url(),
+  void_session_url: zod.string().url(),
+  confirmation_callback_url: zod.string().url().optional(),
+  merchant_label: zod.string(),
+  supported_countries: zod.array(zod.string()),
+  supported_payment_methods: zod.array(zod.string()),
+  test_mode_available: zod.boolean(),
+  subtype: zod.string(),
```

Comment:
is the subtype what we referred to initially as "target"?

Reply:
it should be "target", thanks for flagging this !